### ISSUE TITLE
Specular Texture connected to Roughness Input

### ISSUE THREAD

**Issue by Gnates** [johnzero7/XNALaraMesh#26]
Not sure if this is a mistake.
Isn't the **Specular** texture supposed to connect to the **Specular Input** of the **Principled Shader Node**?
I've seen this happen with most XPS models. 
Let me know if you need more examples.
**XPS Model:** https://www.deviantart.com/chrissy-tee/art/SCVI-Xianghua-XPS-Model-771201341

**Comment by johnzero7** [comment]
They share name, but these are very different concepts.
XPS uses the old pipeline diffuse/spec/bump. PBR pipeline uses albedo/metalic/roughness/bump.
So, none of the textures from XPS is appropriate to use in PBR (except bump, that doesn't change). But the addon just works with what it has.


There are 2 tipes of materials: Metallic and Dielectric. Dielectric means not electrical conductor, so to simplify: Metallic materials are metals (iron, gold, cooper, etc.) and Dielectric is anything that is not metallic (cloth, platstic, wood, etc.)

Has describe in the documentation of the principaled node:
https://docs.blender.org/manual/en/latest/render/cycles/nodes/types/shaders/principled.html
Amount of dielectric specular reflection

Why do I care if a material is Metallic or Dielectric??
Because in PBR (Phisically based rendering) this is one of the properties that affect how a material looks.

In dielectric materials the specular reflection is the same color that the light source, but metallic materials tint the specular reflection with the color of the material. 
http://www.manufato.com/wp-content/uploads/2010/10/imagem-35-english.jpg
for most materials a value of .5 is just fine.

Roughness is the input that indicates how much the light scatters in different directions when it bounces.
Imagine a glossy plastic vs a matte finished one.


TL;DR; nope, roughness is correct

**Comment by Gnates** [comment]
Just for clarification, because the XPS software has some graphical limitations (including lack of PBR), the XPS Tools addon is setup this way to help XPS users with their XPS pipeline workflow in Blender? 
If so, does that mean non-XPS users can still use these textures for Blender PBR use?
Or did you mean that these specific textures were originally created/edited in such a way that it would not be accurate in PBR, and they were specifically made for the XPS graphical environment?

I've never used the XPS software so I'm not familiar with its workflow.
